app/templates/browser-support.hbs

```diff
@@ -109,13 +143,15 @@
           <li>Google Chrome</li>
           <li>Mozilla Firefox</li>
           <li>Microsoft Edge</li>
+          <li>Safari</li>
```

Comment:
Based on my undersanding of the RFC, the change for safari will only affect ember@6 so we should not be changing any other section on this page other than adding the ember@6 section 👍

Reply:
yup! apologies, I went too fast here

Reply:
but this is the evergreen section -- so since we're waiting to merge until ember v6, wouldn't it be true that safari is evergreen?

Reply:
so this is the evergreen section for ember@3 back when there was a bigger distinction between the two cases. For each of the newer Ember versions above this we include Safari 12 on both mobile and desktop even though it's not evergreen. 

We could legitimately restructure this page generally to explain the policy first and make sure that each specific version has an evergreen and non-evergreen list 🤔

Reply:
oh! that makes sense. and the h3 vs h2 above (mentioning ember v3) explains that as well!